WordPress/gutenberg
# Update text description of the "Contain text cursor inside block" preference

## What?
Fixes https://github.com/WordPress/gutenberg/issues/57351

This PR updates the description of the "Contain text cursor inside block" preference in the Accessibility options.

## Why?
The previous description implied the feature was primarily beneficial for screen reader users, which was misleading. The feature impacts all keyboard users, providing better control while navigating between blocks. The new description is clearer, more user-focused, and emphasizes its broader applicability.

## How?
The description was rewritten to explain the functionality in simpler terms that resonate with casual users, outlining how it affects keyboard navigation and enhances usability.

## Testing Instructions

1. Open the WordPress admin dashboard.
2. Navigate to Options > Preferences > Accessibility.
3. Locate the "Contain text cursor inside block" option.
4. Verify the updated description is displayed correctly.
5. Confirm the description accurately explains the behaviour by testing the preference in a post or page.

### Testing Instructions for Keyboard


## Screenshots or screencast 



## Before
![before_preference](https://github.com/user-attachments/assets/6619f86b-74af-49db-9a6a-05301ea71426)


## After
 ![after_preference](https://github.com/user-attachments/assets/4ad59e4e-5f35-4b55-94b4-f3daa5acf9c3)
